tests/integration/tui/test_logs.py

```diff
@@ -101,7 +101,7 @@ async def _wait_log_loaded(tries: int = 25, delay: float = 0.1):
             delay: The delay between retries.
 
         """
-        nonlocal before, count
+        nonlocal before
```

Comment:
This `nonlocal before` is needed.

Warning, don't remove `nonlocal` or `global` statements unless flake8 tells you to, it can break things.